module/components/AppBar/Toolbar.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /**
+     * Useful to extend the style applied to components.
+     */
+    className: PropTypes.string,
+    /**
+     * The content of the component.
```

Comment:
need to add disableGutters as well.

Reply:
Didn't add the additional props as it was a sample component. Anyway I have added it now.